### PR TITLE
Add Common Methods section

### LINKED ISSUE #73: Move meta-analysis section to "tutorials"
Also, would be cool to use NiMARE for this

### LINKED ISSUE #75: Remove just-the-docs page from sidebar
I understand why we would want to retain the documentation, but I don't think we need to render it on the website.

### PR DESCRIPTION
**Pull request by tsalo**
Changes proposed:

- Add Common Methods section. Move the meta-analysis section into this. Closes #73 (though I didn't put it in "tutorials").
- Remove rendered just-the-docs page. The documentation is still in the repo. Closes #75.
- Rename archive section to "Archived Documentation"
- Remove duplicate page on updating the website.

### CODE REVIEW

File: docs/Archive/cubic-X11GUI.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
 layout: default
 title: CUBIC-X11GUI
-parent: archive
+parent: Archived Documentation
```
Comment (kahinimehta) [inline docs/Archive/cubic-X11GUI.md:4]: are you able to rename the folder? I am not

**Review by kahinimehta — APPROVED** [review]

File: docs/Archive/cubic-X11GUI.md (hunk 1 above)
Comment (tsalo) [reply docs/Archive/cubic-X11GUI.md:4]: I can give it a try

Comment (tsalo) [reply docs/Archive/cubic-X11GUI.md:4]: I'm fine with `Archive` at this point though. As long as all of the archived files are in the same folder.

Comment (kahinimehta) [reply docs/Archive/cubic-X11GUI.md:4]: So they appear to be under the same heading on the webpage, but there are two different folders internally

Comment (tsalo) [reply docs/Archive/cubic-X11GUI.md:4]: Should be good now.

**Pull request merged**